src/classes/Passage.ts

```diff
@@ -1,58 +1,63 @@
-import { decodeProtectedHeader, jwtVerify, createRemoteJWKSet } from 'jose';
 import { AuthStrategy } from '../types/AuthStrategy';
 import { PassageConfig } from '../types/PassageConfig';
 import { PassageError } from './PassageError';
-import User from './User';
-import {
-    AppInfo,
-    AppsApi,
-    Configuration,
-    CreateMagicLinkRequest,
-    MagicLink,
-    MagicLinksApi,
-    ResponseError,
-} from '../generated';
+import { AppInfo, AppsApi, Configuration, CreateMagicLinkRequest, MagicLink, ResponseError } from '../generated';
 import apiConfiguration from '../utils/apiConfiguration';
 import { IncomingMessage } from 'http';
 import { getHeaderFromRequest } from '../utils/getHeader';
+import { PassageInstanceConfig } from './PassageBase';
+import { Auth } from './Auth';
+import { User } from './User';
 
 /**
  * Passage Class
  */
 export class Passage {
-    appID: string;
+    private appID: string;
     #apiKey: string | undefined;
```

Comment:
was this and `private _apiConfiguration` kept as-is since they're being removed in the major release?

Reply:
Yeah they're needed for the deprecated functions, they'll be killed off in the major bump